reference/3.0/Microsoft.PowerShell.Core/About/about_Remote_Disconnected_Sessions.md

```diff
@@ -14,34 +14,31 @@ Explains how to disconnect from and reconnect to a PSSession
 
 ## Long Description
 
-Beginning in Windows PowerShell 3.0, you can disconnect from
-a PSSession and reconnect to the PSSession at a later time on
-the same computer or a different computer. The session state
-is maintained and commands in the PSSession continue to
-run while the session is disconnected.
+Beginning in Windows PowerShell 3.0, you can disconnect from a PSSession and
```

Comment:
I don't know the answer to that for sure. But since this is a "remoting" article, I left that as is. This will need to be reviewed and updated for the v6 release.